e2e-tests/cypress/e2e/pages/your-business/index.js

```diff
@@ -0,0 +1,7 @@
+import companyDetails from './companies-house-search/companyDetails';
+import yourBusinessSummaryList from './companies-house-search/yourBusinessSummaryList';
+
+export {
+  companyDetails,
+  yourBusinessSummaryList,
```

Comment:
Should `yourBusinessSummaryList` be a partial instead of in pages? Also maybe could be called `companySummaryList` partial, then easy to user for any company (for e.g exporter or a buyer), in other areas moving forwards. What do you think?

Reply:
done